build.sbt

```diff
@@ -224,8 +222,18 @@ lazy val server: Project = (project in file("server"))
       ),
       publishLocal in Docker := (publishLocal in Docker)
         .dependsOn(yarnTask.toTask(" build"))
-        .dependsOn(createBuildx)
         .value,
+      publish in Docker := {
+        val _ = publishLocal.value
```

Comment:
try: `(publishLocal in Docker).value`

Reply:
It works, I had to fix appropriate directory also.